faucet.go

```diff
@@ -131,60 +60,15 @@ var (
 	// OpenChannelAction represents an action to open channel on post forms
 	OpenChannelAction = "openchannel"
 
-	// lastGeneratedInvoiceTime stores the last time an invoice generation was
-	// attempted.
-	lastGeneratedInvoiceTime time.Time
+	// requestIPs stores the last time an ip did an action,
+	// and is protected by a mutex that must be held for reads/writes.
+	requestIPs map[string]time.Time
 
-	// lastPayInvoiceTime stores the last time an invoice payment was
-	// attempted.
-	lastPayInvoiceTime time.Time
+	// rateLimitMtx is a mutex to protect goroutines from reads/writes
+	// simutaneously.
```

Comment:
You can assume that everyone reading the code will understand (or can search for) what a mutex is. It's more important to inform _what this particular mutex protects_ (as opposed to what a mutex is). 

One convention you can use is to group the members protected by a particular mutex together, so that it's obvious what's happening.

For example, see: https://github.com/decred/dcrd/blob/52d2d2db294b4d9b01f43d69b6e847dd88233164/mining.go#L2082. Notice how the `BgBlkTmplGenerator` has multiple mutexes, each protecting a subset of the members and how they're documented.